src/main/java/se/vandmo/dependencylock/maven/mojos/AbstractDependencyLockMojo.java

```diff
@@ -26,6 +44,12 @@ public abstract class AbstractDependencyLockMojo extends AbstractMojo {
   @Parameter(defaultValue = "${project}", required = true, readonly = true)
   private MavenProject project;
 
+  @Parameter(defaultValue = "${session}", required = true, readonly = true)
+  private MavenSession mavenSession;
+
+  @Parameter(property = "dependencyLock.alsoLockBuild")
+  private boolean alsoLockBuild;
```

Comment:
I think `lockBuild` would be enough? And maybe it should be moved to LockMojo.java ?
I think that the check goal should check the build lock if it exists in the lock file.

Reply:
Indeed.
Reworked that in two phases:

1. Renaming the parameter (https://github.com/vandmo/dependency-lock-maven-plugin/pull/85/commits/991ae4450348f4214e3c463d6a3e1e08cf9e3ff6)
2. Pull down the parameter and make sure the check logic automatically detects if it needs to check build logics (in https://github.com/vandmo/dependency-lock-maven-plugin/pull/85/commits/9a417a4ceed44f743e5faecf8b522a519b4e1c0a)